hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/volume/CreateVolumeHandler.java

```diff
@@ -40,10 +41,15 @@
       description = "Owner of of the volume")
   private String ownerName;
 
+  @Option(names = {"--spaceQuota", "-sq"},
+      description =
+          "Quota Bytes of the newly created volume (eg. 1GB)")
+  private String storagespaceQuota;
+
   @Option(names = {"--quota", "-q"},
       description =
-          "Quota of the newly created volume (eg. 1G)")
-  private String quota;
+          "Quota count of the newly created volume (eg. 5)")
```

Comment:
Correct me if I miss something.
Would it be better to update the description to "Bucket counts of the newly created volume (eg. 5) " ?